book/10-git-internals/1-git-internals.asc

```diff
@@ -1,19 +1,13 @@
-We found that understanding this information was fundamentally important to appreciating how useful and powerful Git is, but others have argued to us that it can be confusing and unnecessarily complex for beginners.
-Thus, we've made this discussion the last chapter in the book so you could read it early or later in your learning process.
-We leave it up to you to decide.
+Maari mong nalaktawan ang kabanatang ito mula sa mas naunang kabanata, o maari kang nakarating dito pagkatapos mo basahin ng sunod sunod ang aklat hanggang sa puntong ito -- sa ano mang kaso, dito tatalakayin ang mga panloob na pagtatrabaho at implementasyon ng Git. Nalaman namin na ang pag-intindi ng impormasyong ito ay mahalaga sa panimula upang mapahalagahan kung gaano kapaki-pakinabang at makapangyarihan ang Git, ngunit ang iba ay nakipagtalo sa amin na ito ay maaaring nakalilito at hindi kinakailangan kumplikado para sa mga nagsisimula. Kaya, ginawa namin ang dikusyong ito bilang huling kabanata sa aklat na ito upang mabasa mo ito nang maaga o huli sa proseso ng iyong pag-aaral. Iniwan na namin sa inyo ang pagpasya.
 
-Now that you're here, let's get started.
-First, if it isn't yet clear, Git is fundamentally a content-addressable filesystem with a VCS user interface written on top of it.
-You'll learn more about what this means in a bit.
+Ngayon na nandito ka na, simulan na natin. Una, kung hindi pa ito klaro sa iyo, ang Git sa panimula ay isang filesystem na maaring i-address ang nilalaman at mayroong VCS na interface para sa mga gumagamit. Malalaman mo ang kahulugan nito sa ilang sandali.
 
-In the early days of Git (mostly pre 1.5), the user interface was much more complex because it emphasized this filesystem rather than a polished VCS.
-In the last few years, the UI has been refined until it's as clean and easy to use as any system out there; however, the stereotype lingers about the early Git UI that was complex and difficult to learn.
+Sa mga naunang araw ng Git (halos bago ang 1.5), ang interface ng gumagamit ay mas kumplikado dahil binigyang diin nito ang filesystem sa halip na pakinisin ang VCS. Sa mga nakaraang taon, ang UI ay mas pinainam hanggang sa ito ay maging malinis at madaling gamitin tulad ng anumang mga sistema; subalit, ang gasgas tungkol sa naunang Git UI na kumplikado at mahirap pag-aralan ay nanatili.
 
-The content-addressable filesystem layer is amazingly cool, so we'll cover that first in this chapter; then, you'll learn about the transport mechanisms and the repository maintenance tasks that you may eventually have to deal with.
+Ang suson ng filesystem na maaring i-address ang nilalaman ay nakamamangha, kung kaya't tatalakayin natin yan ng una sa kabanatang ito; saka, matututuhan mo ang tungkol sa mga mekanismo ng transportasyon at ang mga gawain sa pagpapanatili ng imbakan na maari mong tingnan.
```

Comment:
Ang suson ng filesystem to Ang nilalaman-tinumbok na filesystem

Reply:
maari  to maaari

Reply:
Done. Changed the translation based on the comments